lib/utils/HTTP/get_request.dart

```
@@ -0,0 +1,30 @@
+import 'dart:io';
+
+import 'package:exdock_backend_client/utils/HTTP/HttpData.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:http/http.dart' as http;
+
+Future<HttpData> standardGetRequest(String endpoint) async {
+  FlutterSecureStorage storage = FlutterSecureStorage();
+  int statusCode = 404;
+  String responseBody = "";
+  String? token = await storage.read(key: "jwt");
+
+  // TODO: add server from config
+  Uri uri = Uri.http("127.0.0.1", endpoint);
+  if (token == null) {
+    return HttpData(statusCode: 401, responseBody: null);
+  }
+
+  try {
+    http.Response response = await http.get(
+      uri,
+      headers: {HttpHeaders.authorizationHeader: 'Bearer $token'},
+    );
+
+    statusCode = response.statusCode;
+    responseBody = String.fromCharCodes(response.bodyBytes);
+  } catch (_) {}
```

Should probably throw. So remove the try {} catch {}. Otherwise something needs to be done with the error, because there is no way of knowing now what went wrong.

But my preference is to just let it throw.